Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -1228,6 +1228,16 @@ extension DocumentSnapshot {
     return Position(line: zeroBasedLine, utf16index: utf16Column)
   }
 
+  /// Converts the given `String.Index` to a UTF-16-based line:column position.
+  ///
+  /// If the index does not refer to a valid position within the snapshot, returns the closest valid
+  /// position and logs a fault containing the file and line of the caller (from `callerFile` and
+  /// `callerLine`).
+  func position(of index: String.Index, fromLine: Int = 0) -> Position {
```

Comment:
I don’t think the comment is correct because there’s no `callerFile` and `callerLine` comment. Also, no fault is logged by `LineTable.lineAndUTF16ColumnOf`.